code/drasil-example/swhs/lib/Drasil/SWHS/Body.hs

```diff
@@ -314,8 +314,8 @@ sysCntxtDesc pro = foldlSP [refS sysCntxtFig, S "shows the" +:+.
   S "flow between the", phraseNP (system `andIts` environment)]
 
 sysCntxtFig :: LabelledContent
-sysCntxtFig = llcc (makeFigRef "SysCon") $ fig (foldlSent_
-  [refS sysCntxtFig +: EmptyS, titleize sysCont])
+sysCntxtFig = llcc (makeFigRef "SysCon") 
+  $ fig (titleize sysCont)
```

Comment:
An oddity: SWHS and SWHSNoPCM were the only examples that had selferences in their System Context Figures. This should probably be done at the level of the Figure renderer rather here.

Reply:
agreed!